miden/src/cli/prove.rs

```diff
@@ -26,14 +22,17 @@ pub struct ProveCmd {
     /// Path to proof file
     #[structopt(short = "p", long = "proof", parse(from_os_str))]
     proof_file: Option<PathBuf>,
+    /// Security level for execution proofs generated by the VM
+    #[structopt(short = "s", long = "security", default_value = "96bits")]
+    security: String,
```

Comment:
It's unclear how the user can select RPO here. RPO is far slower than blake to generate proofs, but I think the idea is to leave the option available, regardless.

Reply:
We are not exposing this functionality through the CLI yet. There are a couple of ways to do it (e.g., see https://github.com/0xPolygonMiden/miden-vm/pull/716#discussion_r1115849045)  and I'd rather do it in a different PR.